### DISCUSSION TITLE
data-engineering/2023/05/28/how-to-use-snowflake/

### DISCUSSION THREAD

**Discussion by giscus** [zzsza/blog-comments#143] [Comments]
# data-engineering/2023/05/28/how-to-use-snowflake/

 Data Warehouse 중 하나인 Snowflake 사용법에 대해 작성한 글입니다 Snowflake를 전혀 모르는 분들을 대상으로 설명하기 위한 첫 글이고, 큰 그림 위주로 작성했습니다. 추후에 다른 글도 작성할 예정입니다 2023년에 기본 웹 인터페이스로 변경된 Snowsight 화면을 기준으로 작성되었습니다 

https://zzsza.github.io/data-engineering/2023/05/28/how-to-use-snowflake/

**Comment by ghost** [comment]
단비같은 글입니다. 유데미에서 인도강사분 강의 듣다가 이거보니 이해가 더 되네요. 감사합니다!

**Reply by zzsza** [reply]
앗 제가 댓글을 이제 봤네요..!!! 유데미 강의 듣다가 보니 이해가 더 된다니 뿌듯하네요. 댓글 남겨주셔서 감사합니다!
